plugin/hooks.go

```diff
@@ -133,6 +134,9 @@ type Hooks interface {
 	// UserHasLoggedIn is invoked after a user has logged in.
 	UserHasLoggedIn(c *Context, user *model.User)
 
+	//UserWasCreated is invoked after a user was created.
```

Comment:
Could you please document the required server version?

```suggestion
	// UserWasCreated is invoked after a user was created.
	//
	// Minimum server version: 5.10
```

Reply:
Done.